runtime/js/11_timers.js

```diff
@@ -442,7 +442,10 @@
     if ("function" === typeof callback) {
       callback();
     } else {
-      eval(callback);
+      const result = core.evalContext(callback);
+      if (result[1] != null) {
+        throw result[1].thrown;
```

Comment:
Would `(0, eval)(callback)` work? Rethrowing means this `throw` statement shows up as the exception's origin rather than the evaluated code.

Reply:
I didn't know about `globalThis.eval` vs `eval`. Fixed.